### ISSUE TITLE
Environment issues

### ISSUE THREAD

**Issue by minhnimble** [abdulrahim46/Nimble#10]
It was interesting to see you actually setup Github Action CI to build the project and run tests after each merge. Now as per our our API [documentation](https://github.com/nimblehq/nimble-survey-web/wiki/REST-API#ios), there are more environments that our API should be capable of of supporting, which is

**Comment by abdulrahim46** [comment]
Yes, we can handle this by adding Configurations, storing the configurations in info.plist and defining schemes(Staging & Production) for our project. After that we can setup the scheme in the .yml in github actions section after every pull request.
CI can help us after changes are made by creating a build and running automated tests against the build.

> At the same time, it would be great if you could update the project implementation to handle both environments and point to the correct base endpoint for each environment as well. What do you think?

Yes, I think adding configuration for switching different envirnoment will help us in that.

**Comment by minhnimble** [comment]
Sounds good then, please go ahead and implement the new changes using proper gitflow 😁
